docs/building/windows-instructions.md

```diff
@@ -56,8 +56,8 @@ You can use the Developer Command Prompt, Powershell or work in any regular cmd.
 From a (non-admin) Command Prompt window:
 
 - `build.cmd` - builds the assemblies
-- `build.cmd -runTests` - called after a normal "build.cmd" will run all tests
-- `build.cmd -buildPackages` called after a normal “build.cmd” will create the NuGet packages with the assemblies in “bin"
+- `build.cmd -test` - called after a normal "build.cmd" will run all tests
+- `build.cmd -pack` called after a normal “build.cmd” will create the NuGet packages with the assemblies in `artifacts\packages`"
```

Comment:
This still builds the native components. The command really builds AND tests. Can you please rephrase to reflect that? Same thing for -pack too.